L2 Understading electric vehicless/How do Li-ion batteries in BEVs work.md

```diff
@@ -0,0 +1,157 @@
+
+-=- Source: [U.S. Department of Energy](https://www.energy.gov/science/doe-explainsbatteries)
+
+1. A discharge circuit is formed between the anode and the cathode.
+2. Lithium ions stored in the anode move to the cathode.
+3. Energy is used.
+
+# Important points to remember
+
+- Lithium-ion chemistry prefers partial discharge to deep discharge, so it's best to avoid taking the battery all the way down to zero. Since lithium-ion chemistry does not have a "memory", you do not harm the battery pack with a partial discharge. 
+
+- If the voltage of a lithium-ion cell drops below a certain level, it's ruined.
+- Lithium-ion batteries do not age well. They only last two to three years, even if they are sitting on a shelf unused. So do not "avoid using" the battery with the thought that the battery pack will last five years. It won't. Also, if you are buying a new battery pack, you want to make sure it really is new. 
+
+- If it has been sitting on a shelf in the store for a year, it won't last very long. Manufacturing dates are important.
+
+- Avoid heat, which degrades the batteries.
+
+- Electrical current is measured in amperes, or simply amps. But the impact of one ampere of current from a small battery cell is quite different from the impact of one ampere of current from a large battery cell. Hence a relative measure of electrical current is used to scale the size of a battery cell, and this relative measure is known as the **C rate**. 
+     
+     * The C rate of a battery cell is the level of constant current charge or discharge that the cell can sustain for one hour of time. 
+
+     * For example, a cell having a capacity of ` 20 ampere hours (or 20AH)`  should be able to deliver 20 amperes of current for one hour or two amperes of current for 10 hours. 
+       
+       `20 Ampere-Hour` = `20 Amperes` x `1 hour` **or** `2 Amperes` x `10
+hours`
+
+       This relationship is not a completely linear relationship. It's not quite that simple, but it's a pretty close relationship, 
+
+       So when we say that the cell should be able to deliver 20 amperes of current for one hour, we are saying that the C rate of a 20 ampere hour cell is 20 amperes. It's what we call the 1C rate. In this example, two amperes is what we would call the C divided by 10 rate because it is one-tenth of the 1C rate. 
+
+       We can denote the above statement with the following formula: 
+      
+        `Battery capacity x C rating = Output current`
+
+#### Let's take an example to understand C-rating better. Imagine you have selected a battery with a rating of 2600Mah. Use this PDF for reference 
+[INR18650-25R-Samsung Datasheet](https://www.pupilfirst.school/markdown_attachments/3616/qkvyUQpXzNZfSCXh6M_2UQ)
+
+
+Now, for the battery we have selected, imagine that the maximum current the battery can produce is around 20 Amperes.  can also be written as 20,000 milli-amperes *(from Page 3, point no 3.7)*
```

Comment:
```suggestion
Now, for the battery we have selected, imagine that the maximum current the battery can produce is around 20 amperes.  This can also be written as 20,000 milli-amperes *(from Page 3, point no 3.7)*
```